### PR TITLE
Implement bottom sheet for dm user

### PR DESCRIPTION
**Pull request by cuddle-bunny**
This commit implements bottom sheet for dm user

### CODE REVIEW

File: lib/ui/view/dm_user/.vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (isaacadariku) [inline lib/ui/view/dm_user/.vscode/launch.json:1]: Kindly remove the vscode commit, you can use it locally on your PC

File: lib/ui/view/dm_user/dm_user_viewmodel.dart [hunk 2]
```diff
@@ -74,4 +74,93 @@ class DmUserViewModel extends BaseViewModel {
     );
     notifyListeners();
   }
+
+  Future<dynamic> showButtomSheet(BuildContext context) {
```
Comment (isaacadariku) [inline lib/ui/view/dm_user/dm_user_viewmodel.dart:78]: Make use of `stacked_bottomSheet` service, and register a custom bottomSheet, if you don't know about it, read about it [here](https://pub.dev/packages/stacked_services#bottomsheet-service).

and ensure no material or Cupertino widget is imported in the viewmodel

**Review by isaacadariku — CHANGES_REQUESTED** [review]
Make the changes and ask for a review again

File: lib/ui/view/dm_user/dm_user_viewmodel.dart (hunk 2 above)
Comment (cuddle-bunny) [reply lib/ui/view/dm_user/dm_user_viewmodel.dart]: Done

File: lib/ui/view/dm_user/.vscode/launch.json (hunk 1 above)
Comment (cuddle-bunny) [reply lib/ui/view/dm_user/.vscode/launch.json]: Done

File: .vscode/launch.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (Blazebrain) [inline .vscode/launch.json:1]: add this to your gitignore file so it doesn't push to github

**Review by Blazebrain — CHANGES_REQUESTED** [review]

**Pull request closed without merging**